PINCache/PINDiskCache.m

```diff
@@ -1120,11 +1134,17 @@ - (instancetype)init
 + (instancetype)start
 {
     PINBackgroundTask *task = [[self alloc] init];
```

Comment:
@garrettmoon You think we should just return nil here and not allocate any PINBackgroundTask if isAppExtension == YES?